app/helpers/blacklight/facets_helper_behavior.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 module Blacklight::FacetsHelperBehavior
```

Comment:
It's a little hard to tell from the diff, but as a result of this PR, the only non-deprecated helpers are:

- `has_facet_values?` (probably not long for this world)
- `render_facet_partials` (perhaps part of a future component)
- `render_facet_limit` (possibly obsolete once components get traction)
- `facet_field_in_params`  (a little tricky to deprecate at the moment)
- `facet_field_presenter`